src/test/kotlin/no/nav/tpts/mottak/health/HealthRoutesTest.kt

```diff
@@ -2,7 +2,7 @@ package no.nav.tpts.mottak.health
 
 import io.ktor.http.HttpMethod
 import io.ktor.http.HttpStatusCode
-import io.ktor.routing.routing
+import io.ktor.server.routing.routing
 import io.ktor.server.testing.handleRequest
 import io.ktor.server.testing.withTestApplication
```

Comment:
Står i [migration-guiden](https://ktor.io/docs/eap/migrating-2.html#testing-api) at `withTestApplication` har blitt erstattet med `testApplication`. Litt rart at dette funker i hele tatt, det fremstilles som om `withTestApplication` har blitt fjernet helt 🤔  Foreslår det byttes i denne PR-en evt lages en ny task på det uansett

Reply:
Ja, det er det som gjenstår i oppgraderingen. `withTestApplication` er foreløpig `@Deprecated`. Jeg kan fikse det før merge

Reply:
@tu55eladd har satt noen tester som `@Disabled`, trenger råd. F.eks. `SoknadRoutesTest.should return 404 when soknad not found`

Her kastes en exception når man kaller `client.get()` med statuskoden pakket inn i excpetion. Det virker jo som underlig oppførsel. Forslag til løsning?

Reply:
Endepunktet som ble testet der er jo fjernet så da kan egentlig bare slette hele testen. Mulig Ktor gjør at man kan velge selv hvordan man skal håndtere endepunkt som ikke finnes, er ikke alltid man ønsker å gjøre 404 feks